test/e2e/zookeepercluster_test.go

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package e2e
+
+import (
+	"testing"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	apis "github.com/pravega/zookeeper-operator/pkg/apis"
+	operator "github.com/pravega/zookeeper-operator/pkg/apis/zookeeper/v1beta1"
+	zk_e2eutil "github.com/pravega/zookeeper-operator/pkg/test/e2e/e2eutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestZookeeperCluster(t *testing.T) {
+	pravegaClusterList := &operator.ZookeeperClusterList{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ZookeeperCluster",
+			APIVersion: "pravega.pravega.io/v1beta1",
```

Comment:
it should be `zookeeper.pravega.io/v1beta1`

Reply:
done